os/kernel/binary_manager/binary_manager_load.c

```diff
@@ -544,27 +515,34 @@ static int reloading_thread(int argc, char *argv[])
 		bmdbg("Terminate binary %d\n", bidx);
 	}
 
-	/* 2. Unload common binary */
+	/* Unload common binary */
 	ret = binary_manager_terminate_binary(BM_CMNLIB_IDX);
 	if (ret != OK) {
 		bmdbg("Failed to terminate common binary\n");
 		return BINMGR_OPERATION_FAIL;
 	}
+#else
+	load_cmd = LOADCMD_LOAD;
```

Comment:
What's purpose of this?

Reply:
A loader is created and do something according to `loading type` like LOADCMD_LOAD, LOADCMD_LOAD_ALL.
If CONFIG_SUPPORT_COMMON_BINARY is enabled, we set `LOADCMD_LOAD_ALL` to load all binaries after termination of all.
Otherwise, we set `LOADCMD_LOAD` to load a faulty binary.